setup.cfg

```diff
@@ -1,3 +1,4 @@
 [flake8]
 max-line-length = 105
 extend-ignore = E203, W503
+exclude = */migrations/*
```

Comment:
Why?

Reply:
@kilemensi Since migrations are auto-generated, I don’t think we need to enforce flake8 rules on them.

Reply:
Not sure I follow @esirK ... They might be auto generated but to a formatter/linter they're just another python file. Why not format/lint them as well?

Reply:
We are formating them @kilemensi . This is for `flake8`. I don't think the formatter auto formats for conditions like `line length`. If a migration file length is more than `105`, flake8 will fail which I don't think it shoudl.

Reply:
I could be missing something here @esirK but:

If we are formatting (using `black` which [understands](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8) `flake8` rules), `flake8` shouldn't complain anymore.

The only two possible scenarios I can think of is either we're not formatting at all before linting or the formatter is not set up correctly.

https://www.pantsbuild.org/docs/python-linters-and-formatters

Reply:
The issue is/was by default black [wasn't formatting single string longer than tine length limit ](https://github.com/psf/black/issues/1802)
I've fixed it using `args = ["--preview"]`